docs/source/getting_started/getting_started.md

```diff
@@ -423,7 +390,269 @@ curl -X POST "$URL/users/u1234/personalized_documents" \
+
+curl -X POST "$URL/users/u1234/personalized_documents" \
+    --header "authorizationToken: $FRONTOFFICE_TOKEN" \
+    --header "Content-Type: application/json" \
+    --data '{
+        "filter": {
+            "publication_date": {
+                "$gte": "2022-12-30T11:30:32Z"
+            }
+        },
+        "include_properties": true
+    }'
+```
+
+# Documents with multiple Snippets
+
+Documents can have multiple snippets, and at the core our search and recommendation system is based on the searching for and recommending snippets not documents.
+
+When using documents with multiple snippets you need to be aware about a few things:
+
+- there is a snippet id for each specific snippet a document has
+- we search for snippets not documents, so results can contains multiple (different) snippets of the same document
+- many APIs which accepts either a document id for referring to a document as a whole or a snippet id to refer to a specific snippet
+    - some APIs currently fall back to using the first snippet if only document id is used, but this is intended to change in the future
+
+We made sure that all search APIs in which you can refer to a specific snippet also work with just the document id even if the document has multiple snippets. _But_ the results might not be quite as expected as there can be a huge difference in the search results.
+
+## Snippet Id
+
+Snippets are identified by a snippet id consisting of both the document id and a sub id.
+
+Each document is guaranteed to have a snippet with sub id equals `0`. But besides that
+there are no guarantees about the value of the `sub_id`. E.g. a document might have snippet
+id's with the sub ids `0`, `400`, `2334`.
+
+For example if we ingest a document like
+
+```json
+{
+    "id": "12345",
+    "snippet": "some text"
+},
+```
+
+It would have a single snippet with the id `{ "document_id": "12345", "sub_id": 0 }`.
+
+So if we have following semantic search example
+
+```bash
+curl -X POST "$URL/semantic_search" \
+    --header "authorizationToken: $FRONTOFFICE_TOKEN" \
+    --header "Content-Type: application/json" \
+    --data '{
+        "document": { "id": "12345" },
+        "include_properties": true
+    }'
+```
+
+we could also use the snippet id instead like shown below
+
+```bash
+curl -X POST "$URL/semantic_search" \
+    --header "authorizationToken: $FRONTOFFICE_TOKEN" \
+    --header "Content-Type: application/json" \
+    --data '{
+        "document": { "id": { "document_id": "12345", "sub_id": 0 },
+        "include_properties": true
+    }'
+```
+
+This pattern of replacing document id string with a snippet id struct is also used in all other places
+a snippet id can be used.
+
+Be aware that using `{ "id": "12345" }` represents the whole document but `{ "document_id": "12345", "sub_id": 0 }` represents the first snippet specifically and for some APIs this makes a difference
+for documents which have multiple embeddings as described in the sections below.
```

Comment:
```suggestion
As already stated before the document id represent the whole document while the snippet id represents one of multiple snippets of a document.
The equivalence between document id and a snippet with `sub_id = 0` can change in the future so we suggest not relying on it.
```

I don´t want to encourage them to use the equivalence. or to blindly use document id instead of snippet id. They should choose what they need depending on the use-case.

Reply:
> The equivalence between document id and a snippet with `sub_id = 0` can change in the future so we suggest not relying on it.

This isn't quite correct, even in the current system  in most places they are not equivalent.

> This pattern of replacing document id string with a snippet id struct is also used in all other places
a snippet id can be used.

> I don´t want to encourage them to use the equivalence.

This was more meant to point out that you:

- can migrate from one use-case to the other fluently
- don't have to worry too much about it when you just start out
- isn't an issue when you don't have split documents

I rephrased this chapter. I resolved the two related change requests above as the rephrasing involves that code.